x-pack/elastic-agent/pkg/agent/application/global_config.go

```diff
@@ -8,33 +8,64 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/elastic/beats/v7/libbeat/common"
 	"github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/agent/errors"
 	"github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/config"
 )
 
 var (
-	homePath string
-	dataPath string
+	homePath   string
+	dataPath   string
+	overwrites *common.Config
 )
 
 func init() {
 	homePath = retrieveExecutablePath()
 	dataPath = retrieveDataPath()
+	overwrites = common.SettingFlag(nil, "E", "Configuration overwrite")
+	common.ConfigOverwriteFlag(nil, overwrites, "path.home", "path.home", "", "Agent root path")
+	common.ConfigOverwriteFlag(nil, overwrites, "path.data", "path.data", "", "Data path, agent usually look for beats here")
```

Comment:
rename `Data path, agent usually look for beats here` to `Data path contains Agent managed binary")`